ds_judgements_public_ui/templates/judgment/detail.html

```diff
@@ -1,8 +1,8 @@
 {% extends "layouts/judgment.html" %}
 {% block content %}
-  {% if context.judgment %}
+  {% if context.document %}
```

Comment:
Is this a good point to move this from `context['context']['judgment']` to just `context['document']`?